specification/resources/monitoring/create_alert.yml

```
@@ -0,0 +1,32 @@
+operationId: create_alert_policy
+
+summary: Create Alert Policy
```

```suggestion
summary: "[Beta] Create Alert Policy"
```

Might not be a bad idea to prefix these with `[Beta]`. This is the first major beta API since we moved to the new API documentation platform so we don't have a pattern yet. Though this is similar to how the old custom site worked. Adding `product_lifecycle: beta` to an operation there would prefix the operation's title with `Beta`.

---

Good catch - should have done this initially as the generated preview docs were used as the beta docs. The API is entering GA with the merge of this PR, so I don't think this is necessary anymore.